lib/tasks/README.md

```diff
@@ -1,6 +1,20 @@
-Tasks (.rake files) in this directory will be available in the main ManageIQ app.
-They can be executed in the provider gem via the app: namespace
+Tasks (.rake files) in this directory will be available as public tasks in the main
+ManageIQ app. They can be executed in the plugin gem via the app: namespace
 
+```shell
 bin/rails app:<task>
+```
 
-Task private to the provider should go into lib/tasks/tasks_private.
+Since these tasks are public, please namespace them, as in the following example:
+
+```ruby
+namespace "manageiq:automation_engine" do
+  desc "Explaining what the task does"
+  task :your_task do
+    # Task goes here
+  end
+end
+```
+
+Tasks places in the lib/tasks_private directory will be private to the plugin
```

Comment:
Should be `Tasks placed ...`

Reply:
I'll have to change that in the generator and blast it out separately. Let's leave it here for now. I have a couple of other things to blast out in the next round.